source/form.html

```diff
@@ -7,15 +7,15 @@
     <link href="./css/style.css" rel="stylesheet">
   </head>
   <body class="page__body">
-    <header class="page-header container">
+    <header class="page-header page-header--inner container">
```

Comment:
может быть имеет смысл сделать модификатор наоборот - для главной страницы. Внутренних может быть сколько угодно, а главная - одна. Пример для главной страницы:
```suggestion
    <header class="page-header page-header--home container">
```